app/views/admin/products_v3/components/_product_actions.html.haml

```diff
@@ -5,3 +5,4 @@
       = link_to t('admin.products_page.actions.edit'), edit_admin_product_variant_path(product, variant), class: "vertical-ellipsis-menu-content-item"
     - else
       = link_to t('admin.products_page.actions.edit'), edit_admin_product_path(product), class: "vertical-ellipsis-menu-content-item"
+    = link_to t('admin.products_page.actions.clone'), clone_admin_product_path(product), class: "vertical-ellipsis-menu-content-item"
```

Comment:
I can see that this partial is used for two cases: product and variant (there are two types of record that can appear on this screen). 
This new action is only relevant for product, so it shouldn't show for a variant. I think this can be easily achieved simply by indenting to include in the above `else` block:

```suggestion
      = link_to t('admin.products_page.actions.clone'), clone_admin_product_path(product), class: "vertical-ellipsis-menu-content-item"
```